client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -119,6 +119,12 @@ export default function getThankYouPageUrl( {
 			debug( 'returning sanitized internal redirectTo', redirectTo );
 			return sanitizedRedirectTo;
 		}
+
+		if ( hostname === 'cloud.jetpack.com' ) {
```

Comment:
Enable redirection to Jetpack cloud after renewing a subscription from cloud.

Reply:
Could you add a test for this to https://github.com/Automattic/wp-calypso/blob/21f7f7b865bb2aa5a210289115c6ce0f04f71c6d/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js#L41 ?